### ISSUE TITLE
Cannot Compile 'clockdisplay.h'

### ISSUE THREAD

**Issue by Ricky3Nut** [CircuitSetup/Time-Circuits-Display#11]
I keep getting the following error when I try to compile the sketch and program the board. I can see the file in ...software/src/timecircuits and its open in a tab in IDE but for some reason it keeps fizzing out on me.... Any help please?? :-) Thanks!

tc_wifi.h:23:26: fatal error: clockdisplay.h: No such file or directory
compilation terminated.
exit status 1
clockdisplay.h: No such file or directory

**Comment by CircuitSetup** [comment]
I believe with the Arduino IDE you have to have the clockdisplay files in your libraries directory. Have you tried this?

**Comment by Ricky3Nut** [comment]
Yep, that was it. Thanks! I actually had 4 or 5 others that were doing the same thing then I figured out by experimenting just before getting your message. But that helped confirm the problem and solution, so thank you very much! Now I am stuck on getting the ESP32 Sketch Data tool installed so I can upload the sounds. I am following the tutorial you linked but for some reason I cannot locate a Tools folder in my Arduino directory. I had to stop and go sit down for Thanksgiving dinner so I will come back in a bit and see if I can get it sorted....
